package.json

```diff
@@ -8,15 +8,15 @@
   ],
   "homepage": "https://prismatic.io",
   "bugs": {
-    "url": "https://github.com/prismatic-io/prism"
+    "url": "https://github.com/KeyedIn/prism"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/prismatic-io/prism.git"
+    "url": "https://github.com/KeyedIn/prism.git"
```

Comment:
Another thing we need to ensure to update when we change our GitHub from KeyedIn -> KIMCO.